modules/base/rendering/screenspacerenderablerenderable.cpp

```diff
@@ -0,0 +1,294 @@
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/base/rendering/screenspacerenderablerenderable.h>
+
+#include <openspace/camera/camera.h>
+#include <openspace/rendering/renderable.h>
+#include <openspace/scene/rotation.h>
+#include <openspace/scene/scale.h>
+#include <openspace/scene/translation.h>
+#include <openspace/util/updatestructures.h>
+
+namespace {
+    constexpr openspace::properties::Property::PropertyInfo TimeInfo = {
+        "Time",
+        "Time",
+        "The time (in J2000 seconds) that is used to calculate transformations and the "
+        "renderable's data."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraPositionInfo = {
+        "CameraPosition",
+        "Camera Position",
+        "Specifies the location of the virtual camera that is showing the renderable "
+        "class. This position is provided in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraCenterInfo = {
+        "CameraCenter",
+        "Camera Center",
+        "The location of the camera's focal point. The camera's view direction will "
+        "always be pointing at the provided center location. This position is provided "
+        "in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraUpInfo = {
+        "CameraUp",
+        "Camera Up",
+        "The direction that is 'up' for the provided camera. This value does not have "
+        "any units."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraFovInfo = {
+        "CameraFov",
+        "Camera Field-of-view",
+        "The camera's field-of-view in degrees."
```

Comment:
```suggestion
        "Camera Field of View",
        "The camera's field of view in degrees."
```
To match how we've written this elsewhere in user facing descriptions